src/api/detail-meeting/cancelGroup.ts

```
@@ -0,0 +1,30 @@
+			},
+		);
+
+		if (!response.ok) {
+			const errorData = await response.json();
+			throw new Error(errorData.message || `서버 오류: ${response.statusText}`);
+		}
+
+		const data = await response.json();
+		return data;
+	} catch (error) {
+		console.error('서버 응답 상태 코드:', error);
```

ESLint

[error] Unexpected console statement.

(no-console)